docs/tutorials/intro_to_chemistry.ipynb

```diff
@@ -0,0 +1,310 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "Intro To Chemistry Tutorial.ipynb",
+      "provenance": [],
+      "collapsed_sections": []
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "N4327P0aWzVh",
+        "colab_type": "text"
+      },
+      "source": [
+        "## Introduction to Electronic Structure Theory\n",
+        "Electronic Structure Theory describes the motions of electrons in atoms or molecules. Because the electrons are so small, one needs to use quantum mechanics to solve for their motion.\n",
+        "### Schrodinger Equation For Molecules\n",
+        "$$H\\Psi=E\\Psi$$\n",
+        "$$\\Psi = \\Psi(\\vec{r_1}, \\omega_1, \\vec{r_2}, \\omega_2,...) = \\Psi(\\vec{x_1}, \\vec{x_2},...)$$\n",
+        "\n",
+        "* The state of an electron is represented by a wavefunction $\\Psi(\\vec{x})$ where $\\vec{x} = (\\vec{r}, \\omega)$; $\\vec{r} = (x, y, z) = (r, \\theta, \\phi)$ represents the position of the electron and $\\omega$ represents represents the spin of the electron.  \n",
+        "* Note that elementary particles like electron carry an intrinsic form of angular momentum called Spin, which cannot be described by a function of positional variables $\\vec{r}$, and is somewhat analogous to earth rotating around its own axis.\n",
+        "* Since electrons (fermions) are indistinguishable, the wavefunction $\\Psi(\\vec{x_1}, \\vec{x_2}, ...)$ must be anti-symmetric to satisfy Pauli's Exclusion Principle.\n",
+        "\n",
+        "### Molecular Hamiltonian\n",
+        "$$\\hat{H} = \\underset{\\text{Nuclear K.E.}}{\\hat{T}_N(R)} + \\underset{\\text{Electronic K.E.}}{\\hat{T}_e(r)} + \\underset{\\text{Elec/Nuc attraction}}{\\hat{V}_{eN}(r, R)} + \\underset{\\text{Nuc/Nuc repulsion}}{\\hat{V}_{NN}(R)} + \\underset{\\text{Elec/Elec repulsion}}{\\hat{V}_{ee}(r)}$$\n",
+        "\n",
+        "### Born-Oppenheimer Approximation\n",
+        "* Since Nuclei are much larger than electrons, consider the nuclei to be fixed classical point charges. \n",
+        "* Therefore, Nuclear coordinates $R$ become fixed parameters; i.e. $\\hat{T}_N(R) \\approx 0$, $\\hat{V}_{NN}(R)$ is a constant and $\\hat{V}_{eN}(r; R)$ depends only parametrically on $R$.\n",
+        "* This results in Electronic Hamiltonian given as\n",
```

Comment:
'This results in the electronic structure Hamiltonian:'

Reply:
done